src/e2e/resources/data/AdminSearchPageE2ETest_SqlEntities.json

```diff
@@ -1,44 +1,118 @@
 {
   "accounts": {
-    "instructor1OfCourse1": {
-      "id": "00000000-0000-4000-8000-000000000001",
-      "googleId": "tm.e2e.ASearch.instr1",
-      "name": "Instructor1 of Course1",
-      "email": "ASearch.instructor1@gmail.tmt"
+      "instructor1OfCourse1": {
+        "id": "00000000-0000-4000-8000-000000000001",
+        "googleId": "tm.e2e.ASearch.instr1",
+        "name": "Instructor1 of Course1",
+        "email": "ASearch.instructor1@gmail.tmt"
+      },
+      "instructor2OfCourse1": {
+        "id": "00000000-0000-4000-8000-000000000002",
+        "googleId": "tm.e2e.ASearch.instr2",
+        "name": "Instructor2 of Course1",
+        "email": "ASearch.instructor2@gmail.tmt"
+      },
+      "student1InCourse1": {
+        "id": "00000000-0000-4000-8000-000000000003",
+        "googleId": "tm.e2e.ASearch.student1",
+        "name": "Student1 in course1",
+        "email": "ASearch.student@gmail.tmt"
+      }
```

Comment:
_SqlEntities.json are supposed to only contain accounts, accountRequests, notifications, as they are meant to test our beta deployment (refer to: https://github.com/TEAMMATES/teammates/pull/12902#pullrequestreview-1941992081), do move all these course entities into its own file, similar to https://github.com/TEAMMATES/teammates/pull/12902